lib/go-atscfg/ipallowdotconfig.go

```diff
@@ -270,20 +270,32 @@ func MakeIPAllowDotConfig(
 		// order matters, so sort before adding the denys
 		sort.Sort(ipAllowDatas(ipAllowDat))
 
-		// start with a deny for PUSH and PURGE - TODO CDL: parameterize
+		// add deny for PUSH and PURGE - TODO CDL: parameterize
 		if isMid { // Edges already deny PUSH and PURGE
+			// Push gets placed at the start of the file
 			ipAllowDat = append([]ipAllowData{
 				{
 					Src:    `0.0.0.0-255.255.255.255`,
 					Action: ActionDeny,
-					Method: `PUSH|PURGE`,
+					Method: `PUSH`,
 				},
 				{
 					Src:    `::-ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff`,
 					Action: ActionDeny,
-					Method: `PUSH|PURGE`,
+					Method: `PUSH`,
 				},
 			}, ipAllowDat...)
+			// PURGE gets placed at the end of the file
```

Comment:
Unfortunately, this effectively opens PURGE up to every host that can request any data at all from the mid. This isn't likely to be what folks want. Ideally, we parameterize it, but it might work reasonably well to allow it from localhost in the simplest case so folks can use local system access controls to flexibly limit access.